executor/src/main/java/com/redhat/service/smartevents/executor/impl/CloudEventExecutorImpl.java

```diff
@@ -0,0 +1,109 @@
+package com.redhat.service.smartevents.executor.impl;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.smartevents.executor.Executor;
+import com.redhat.service.smartevents.executor.MetricsConstants;
+import com.redhat.service.smartevents.executor.filters.FilterEvaluator;
+import com.redhat.service.smartevents.executor.filters.FilterEvaluatorFactory;
+import com.redhat.service.smartevents.infra.models.dto.ProcessorDTO;
+import com.redhat.service.smartevents.infra.transformations.TransformationEvaluator;
+import com.redhat.service.smartevents.infra.transformations.TransformationEvaluatorFactory;
+import com.redhat.service.smartevents.infra.utils.CloudEventUtils;
+import com.redhat.service.smartevents.processor.actions.ActionInvoker;
+
+import io.cloudevents.CloudEvent;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Timer;
+
+public class CloudEventExecutorImpl implements Executor {
```

Comment:
@kostola / @lucamolteni I think it would be very useful to have a label on the `pod` to help us determine the type of processor that this executor is handling, so that we can do things like:

```
kubectl get pod -l "com.redhat.smartevent/processor-type=sink"
```